documental/models.py

```diff
@@ -5,26 +5,32 @@
 
```

Comment:
Trailing whitespace.

Please use pep8 to format this document.

Reply:
Adjusted.